[2024-01-31T08:04:32.990+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: manage_data_lake_dag.dataproc_job_raw_to_formatted manual__2024-01-31T08:04:07.888594+00:00 [queued]>
[2024-01-31T08:04:32.996+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: manage_data_lake_dag.dataproc_job_raw_to_formatted manual__2024-01-31T08:04:07.888594+00:00 [queued]>
[2024-01-31T08:04:32.996+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 6
[2024-01-31T08:04:33.012+0000] {taskinstance.py:2191} INFO - Executing <Task(DataprocSubmitJobOperator): dataproc_job_raw_to_formatted> on 2024-01-31 08:04:07.888594+00:00
[2024-01-31T08:04:33.019+0000] {standard_task_runner.py:60} INFO - Started process 139711 to run task
[2024-01-31T08:04:33.025+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'manage_data_lake_dag', 'dataproc_job_raw_to_formatted', 'manual__2024-01-31T08:04:07.888594+00:00', '--job-id', '457', '--raw', '--subdir', 'DAGS_FOLDER/data_lake_dag.py', '--cfg-path', '/tmp/tmp0avf7fbh']
[2024-01-31T08:04:33.028+0000] {standard_task_runner.py:88} INFO - Job 457: Subtask dataproc_job_raw_to_formatted
[2024-01-31T08:04:33.074+0000] {task_command.py:423} INFO - Running <TaskInstance: manage_data_lake_dag.dataproc_job_raw_to_formatted manual__2024-01-31T08:04:07.888594+00:00 [running]> on host instance-1.europe-west9-a.c.data-lake-project-409321.internal
[2024-01-31T08:04:33.151+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='juniortemgoua0' AIRFLOW_CTX_DAG_ID='manage_data_lake_dag' AIRFLOW_CTX_TASK_ID='dataproc_job_raw_to_formatted' AIRFLOW_CTX_EXECUTION_DATE='2024-01-31T08:04:07.888594+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-01-31T08:04:07.888594+00:00'
[2024-01-31T08:04:33.153+0000] {dataproc.py:2317} INFO - Submitting job
[2024-01-31T08:04:33.159+0000] {connection.py:234} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2024-01-31T08:04:33.164+0000] {base.py:83} INFO - Using connection ID 'google_cloud_default' for task execution.
[2024-01-31T08:04:33.352+0000] {dataproc.py:2329} INFO - Job 780cef79-fad8-4782-b13d-6a9c00ec9ce3 submitted successfully.
[2024-01-31T08:04:33.365+0000] {dataproc.py:2363} INFO - Waiting for job 780cef79-fad8-4782-b13d-6a9c00ec9ce3 to complete
[2024-01-31T08:06:54.420+0000] {dataproc.py:2367} INFO - Job 780cef79-fad8-4782-b13d-6a9c00ec9ce3 completed successfully.
[2024-01-31T08:06:54.435+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=manage_data_lake_dag, task_id=dataproc_job_raw_to_formatted, execution_date=20240131T080407, start_date=20240131T080432, end_date=20240131T080654
[2024-01-31T08:06:54.461+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-01-31T08:06:54.487+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
